utils/getInvoices/tests/getInvoices.discount.test.ts

```diff
@@ -238,11 +238,13 @@ describe('getInvoices - DISCOUNT', () => {
         company,
       })
 
-      expect(invoices).toContainEqual({
-        type: ServiceType.Discount,
-        price: 1000,
-        sum: 1000,
-      })
+      expect(invoices).toContainEqual(
+        expect.objectContaining({
+          type: ServiceType.Discount,
+          price: 1000,
+          sum: 1000,
+        })
+      )
```

Comment:
No `expect.objectContaining()` on 'shoud load' tests. We should receive exact result